t/rose-task-run/35-app-prune-log-on-hosts-sharing-fs.t

```diff
@@ -55,7 +55,7 @@ run_pass "${TEST_KEY}" \
 
 TEST_KEY="${TEST_KEY_BASE}-prune.log"
 grep \
-    "ssh .* \\(${JOB_HOST_1}\\|${JOB_HOST_2}\\).* ls. -d..* 19700101T0000Z" \
+    "ssh .* \\(${JOB_HOST_1}\\|${JOB_HOST_2}\\).* ls -d.* 19700101T0000Z" \
```

Comment:
I'm not sure what those extra dots did, probably just captured a surplus space which has since been removed?